### DISCUSSION TITLE
Not letting me post any media

### DISCUSSION THREAD

**Discussion by dytractial** [mastodon/mastodon#18083] [Q&A]
My instance grays out the publish button whenever I try to attatch any media. Im new to mastodon so I dont know what is happening, it could be my connection but media on every other platform (discord, twitter, instagram, snapchat) all work fine on my school wi-fi. Im on iOS 15.0.2.

**Comment by nightpool** [comment]
You'll have to contact your instance admin for this, there's no way we can debug this issue without having access to the instance this is happening on.
